src/fnmp.cpp.props

```diff
@@ -1,49 +0,0 @@
-<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <!-- Use debug libraries in debug mode -->
-  <PropertyGroup Condition="'$(Configuration)'=='Debug'" Label="Configuration">
-    <UseDebugLibraries>true</UseDebugLibraries>
-  </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)'=='Release'" Label="Configuration">
-    <UseDebugLibraries>false</UseDebugLibraries>
-  </PropertyGroup>
-  <!-- Enable static analysis during the build -->
-  <PropertyGroup>
-    <RunCodeAnalysis>true</RunCodeAnalysis>
-    <CodeAnalysisTreatWarningsAsErrors>true</CodeAnalysisTreatWarningsAsErrors>
-  </PropertyGroup>
-  <!-- Use the 'build' directory for intermediate files -->
-  <PropertyGroup>
-    <IntDir>$(SolutionDir)build\$(Platform)_$(Configuration)\obj\$(ProjectName)\</IntDir>
-  </PropertyGroup>
-  <!-- OneBranch goop -->
-  <PropertyGroup>
-    <UseInternalMSUniCrtPackage>true</UseInternalMSUniCrtPackage>
-  </PropertyGroup>
-  <!-- Define DBG in Debug configurations -->
-  <ItemDefinitionGroup Condition="'$(Configuration)'=='Debug'">
-    <ClCompile>
-      <PreprocessorDefinitions>DBG;%(PreprocessorDefinitions)</PreprocessorDefinitions>
-    </ClCompile>
-  </ItemDefinitionGroup>
-  <!-- Project-wide compile properties (e.g. defines, includes) -->
-  <ItemDefinitionGroup>
-    <ClCompile>
-      <AdditionalIncludeDirectories>
-        $(SolutionDir)published\external;
-        %(AdditionalIncludeDirectories)
-      </AdditionalIncludeDirectories>
-      <PreprocessorDefinitions>
```

Comment:
I just noticed I didn't include these. Who should have it?

Reply:
This file, or this section? If it builds without this stuff, then it seems fine?

Reply:
This comment was meant to refer to these lines:
```
        POOL_NX_OPTIN_AUTO=1;
        POOL_ZERO_DOWN_LEVEL_SUPPORT=1;
```
I assume they should not be removed?

Reply:
In every project? Or just a selection? Which?